core/src/main/java/org/mobilitydata/gtfsvalidator/annotation/FieldTypeEnum.java

```diff
@@ -0,0 +1,21 @@
+package org.mobilitydata.gtfsvalidator.annotation;
+
+public enum FieldTypeEnum {
+    INTEGER,
+    FLOAT,
```

Comment:
In the spec under "Field Types" there is a "non-negative float" type - do you think that should be explicitly defined here?

Reply:
I see later that you test for these types for this and INTEGER - what you have LGTM

Reply:
Yes, this non negative guy is weird and we bumped into it repeateadly.